src/runtime/types.cpp

```diff
@@ -1542,6 +1542,22 @@ class AttrWrapper : public Box {
         return rtn;
     }
 
+    static Box* clear(Box* _self) {
+        RELEASE_ASSERT(_self->cls == attrwrapper_cls, "");
+        AttrWrapper* self = static_cast<AttrWrapper*>(_self);
+
+        HCAttrs* attrs = self->b->getHCAttrsPtr();
+        RELEASE_ASSERT(attrs->hcls->type == HiddenClass::NORMAL || attrs->hcls->type == HiddenClass::SINGLETON, "");
+
+        while (true) {
+            const auto& attrMap = attrs->hcls->getStrAttrOffsets();
+            if (attrMap.size() == 0)
+                break;
+            self->b->delattr(attrMap.begin()->first(), NULL);
+        }
```

Comment:
I tried this one put I had to make ```appendNewHCAttr``` public and and then it crashed so I left it as is :-D.